src/user/user.service.ts

```diff
@@ -205,7 +205,7 @@ export class UserService {
 
     const isNameOrLanguageUpdated =
       user.signUpLanguage !== updateUserDto.signUpLanguage && user.name !== updateUserDto.name;
-    updateServiceUserProfilesUser(user, isNameOrLanguageUpdated, user.email);
+    await updateServiceUserProfilesUser(user, isNameOrLanguageUpdated, user.email);
```

Comment:
I see why this `await` was added for testing purposes, however we purposely didn't want to await this function as it will increase the response time for creating a user.

There may be another solution to testing without `await`, maybe 
```
mocked.mockImplementationOnce(async () => {
        return new Error('Crisp API call failed')
  })
```

Reply:
Thanks @annarhughes - totally understand your requirement - ive removed the awaits and all tests still pass locally so hopefully it was a blip on my local and CI will be fine 🤞🏼 - I've updated the main description with detail

Reply:
Perfect thanks for the quick update! Approving the PR now 🎉